docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -306,33 +306,22 @@ Replace all `<>` place holders with the real values that we got from all previou
 ...
 ...
 ...
+    const parser = new WMTSCapabilities();
+    const capabilitiesResponse = await fetch('<RASTER-RASTER-SERVING-SERVICE_URL>/wmts/1.0.0/WMTSCapabilities.xml');      // from Step_4
```

Comment:
Why don't you use the build-in capabilities URL from links?

The following should be changed to 
<mc:links scheme="WMTS" name="[desired_layer_identifier]" description=""
url=[capabilities-url]>
</mc:links>
and then reference the link